installer/Migration/V5_1_0/lang-string/recruitment.yaml

```diff
@@ -0,0 +1,64 @@
+langStrings:
+  -
+    value: 'Vacancy'
+    unitId: vacancy
+  -
+    value: 'Hiring Manager'
+    unitId: hiring_manager
+  -
+    value: 'Candidate Name'
+    unitId: candidate_name
+  -
+    value: 'Keywords'
+    unitId: keywords
+  -
+    value: 'Date of Application'
+    unitId: date_of_application
+  -
+    value: 'Method of Application'
+    unitId: method_of_application
+  -
+    value: 'Enter comma se'
+    unitId: enter_comma_se
+  -
+    value: 'Candidate'
```

Comment:
There are translations for the word candidate in most languages. Please check again with the 4x xml files